### PR TITLE
[13.0][MIG] stock_quant_reservation_info

### PR DESCRIPTION
**Pull request by SergiCForgeFlow**
Standard migration from 14.0 to 13.0

### CODE REVIEW

File: stock_quant_reservation_info/views/stock_quant.xml [hunk 1]
```diff
@@ -4,10 +4,11 @@
         <field name="model">stock.quant</field>
         <field name="inherit_id" ref="stock.view_stock_quant_tree_editable" />
         <field name="arch" type="xml">
-            <field name="available_quantity" position="after">
-                <field name="reserved_quantity" optional="show" />
+            <field name="lot_id" optional="show" />
+            <field name="reserved_quantity" position="after" optional="show">
                 <button
                     name="action_reserved_moves"
+                    string="Document Related"
```
Comment (LoisRForgeFlow) [inline stock_quant_reservation_info/views/stock_quant.xml]: Reservation Details

File: stock_quant_reservation_info/views/stock_quant.xml [hunk 2]
```diff
@@ -23,10 +24,11 @@
         <field name="model">stock.quant</field>
         <field name="inherit_id" ref="stock.view_stock_quant_tree" />
         <field name="arch" type="xml">
-            <field name="available_quantity" position="after">
-                <field name="reserved_quantity" optional="show" />
+            <field name="lot_id" optional="show" />
+            <field name="reserved_quantity" position="after" optional="show">
                 <button
                     name="action_reserved_moves"
+                    string="Document Related"
```
Comment (LoisRForgeFlow) [inline stock_quant_reservation_info/views/stock_quant.xml]: Reservation Details

File: stock_quant_reservation_info/views/stock_move_line.xml [hunk 3]
```diff
@@ -8,6 +8,7 @@
                 <field name="picking_id" />
                 <button
                     name="action_view_picking_from_reserved"
+                    string="Document Related"
```
Comment (LoisRForgeFlow) [inline stock_quant_reservation_info/views/stock_move_line.xml]: Related Document

**Review by LoisRForgeFlow — COMMENTED** [review]
Small comments in the wording of the different buttons. Other than that, LGTM, thanks for the test case!

File: stock_quant_reservation_info/views/stock_move_line.xml (hunk 3 above)
Comment (SergiCForgeFlow) [reply stock_quant_reservation_info/views/stock_move_line.xml]: Done!

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM

**Review by DavidJForgeFlow — APPROVED** [review]

**Review by OriolMForgeFlow — APPROVED** [review]

**Pull request merged**